editors/code/rollup.config.js

```diff
@@ -1,24 +1,27 @@
-import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
 import nodeBuiltins from 'builtin-modules';
 
-export default {
-    input: 'out/main.js',
+/** @type { import('rollup').RollupOptions } */
```

Comment:
Oh, this one is very neat!

Reply:
Though, I guess the only thing that is left this one guy as the first line in this file?
```ts
// @ts-check
```

Reply:
In vscode you will get the typing on hover (and completion) without the explicit `// @ts-check`, but it's probably a good idea to add it anyway.